coalib/coala_delete_orig.py

```diff
@@ -9,11 +9,24 @@
 
 
 def main(log_printer=None, section: Section=None):
+    configure_logging()
+    return start_delete_orig(log_printer, section)
+
+
+def start_delete_orig(log_printer=None, section: Section=None):
+    """
+    >>> import logging
+    >>> import sys
+    >>> from coalib.output.Logging import configure_logging
+    >>> configure_logging(stdout=True, color=False)
+    20
+    >>> start_delete_orig(log_printer="")
+    [WARNING][... Using log_printer is deprecated...
+    >>> logging.shutdown()
```

Comment:
1. I think it's not a good example for documentation
2. this belongs to the commit before^^